okteto/install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -e
+
+subdomain="${CLUSTER_NAME}.k8s.civo.com"
+email="$EMAIL"
+adminToken=$ACCESS_KEY
+ingress=$(kubectl get svc -n=kube-system traefik -ojsonpath='{ .spec.clusterIP }')
```

Comment:
@saiyam1814 will the installer have access to do this?

Reply:
This looks fine